packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -759,8 +767,43 @@ class PlatformViewRenderBox extends RenderBox {
     }
   }
 
+  /// How to behave during hit testing.
+  // The implicit setter is enough here as changing this value will just affect
+  // any newly arriving events there's nothing we need to invalidate.
+  PlatformViewHitTestBehavior hitTestBehavior;
+
+  /// {@macro  flutter.rendering.platformView.updateGestureRecognizers}
+  ///
+  /// Any active gesture arena the `PlatformView` participates in is rejected when the
+  /// set of gesture recognizers is changed.
+  void updateGestureRecognizers(Set<Factory<OneSequenceGestureRecognizer>> gestureRecognizers) {
```

Comment:
What do you think about extracting all the common touch logic for PlatformViewRenderBox and AndroidViewRenderBox to a mixin?